### PR TITLE
offhand version-switch/prac bugfix

### PR DESCRIPTION
**Pull request by Anxietie**
fixed bug where offhand item (potentially important item) disappears when switching from versions >1.9 to <1.9
fixed bug where prac plate wouldn't be removed from inventory if it was in offhand

### CODE REVIEW

File: src/main/java/com/renatusnetwork/momentum/utils/Utils.java [hunk 1]
```diff
@@ -218,13 +219,12 @@ private static ItemStack getItemStackIfExists(Inventory inventory, ItemStack sea
 
         ItemStack foundItem = null;
 
-        if (searchItem != null && searchItem.hasItemMeta() && searchItem.getItemMeta().hasDisplayName())
+        if (searchItem != null /*&& searchItem.hasItemMeta() && searchItem.getItemMeta().hasDisplayName()*/)
```
Comment (bprosen) [inline src/main/java/com/renatusnetwork/momentum/utils/Utils.java]: are these comments needed

File: src/main/java/com/renatusnetwork/momentum/data/stats/StatsManager.java [hunk 2]
```diff
@@ -862,10 +863,14 @@ public void resetPracticeDataOnly(PlayerStats playerStats)
         if (playerStats.inPracticeMode())
         {
             Player player = playerStats.getPlayer();
+            PlayerInventory inv = player.getInventory();
             ItemStack item = Utils.getPracPlateIfExists(player.getInventory());
 
-            if (item != null)
-                player.getInventory().remove(item);
+            if (item != null) {
```
Comment (bprosen) [inline src/main/java/com/renatusnetwork/momentum/data/stats/StatsManager.java:869]: while this is good, we are also going to want to do this offhand checking for leave item removal if it doesn't already exist. I believe sword/shield does this so no need to add it there but make sure of that too

File: src/main/java/com/renatusnetwork/momentum/utils/Utils.java (hunk 1 above)
Comment (Anxietie) [reply src/main/java/com/renatusnetwork/momentum/utils/Utils.java]: thought i removed them

File: src/main/java/com/renatusnetwork/momentum/data/stats/StatsManager.java (hunk 2 above)
Comment (Anxietie) [reply src/main/java/com/renatusnetwork/momentum/data/stats/StatsManager.java:869]: leaving a level with redstone dust in offhand gets handled now
leaving a level with another item in offhand (prismarine for example) will keep it in offhand

**Review by bprosen — APPROVED** [review]
nice work

**Pull request merged**